documentation/docs/components/round_button.md

```diff
@@ -0,0 +1,36 @@
+# Round button
+
+## Example
+
+| type        | name                | value                          |parameter_list |
+| ---------   | ------------        | ------                         |--------- |
+|round_button |round_button_par_2_1	|Icon_src parameter empty string |icon_src: |
+|round_button |round_button_par_2_2	|Icon_src parameter string	     |icon_src: plh_images/icons/play.svg|
```

Comment:
Value column can be removed since it is not interpreted by the `round_button` component.
```suggestion
| type        | name                                      |parameter_list |
| ---------   | ------------                         |--------- |
|round_button |round_button_par_2_1	 |icon_src: |
|round_button |round_button_par_2_2	     |icon_src: plh_images/icons/play.svg|
```